### ISSUE TITLE
How to generate IXGBE recipe for ds3617xs_42218 or build ds3617xs_42218u2?

### ISSUE THREAD

**Issue by tlsnine** [pocopico/tinycore-redpill#13]
Hi.

How can I get the ixgbe driver working with DS3617xs?
I get this error `The extension pocopico.ixgbe was found. However, the extension index has no recipe for ds3617xs_42218 platform.` when building.

I also see references to the driver being supported in ds3617xs_42218u2, but I'm not able to build that version with the default script.
Any update on that?

Thanks for all the work!

**Comment by tlsnine** [comment]
Since no one is checking these out, here's an update from my own experience:

For me, it's not just the DS3617xs ixgbe extension that isn't working with my onboard network.
It seems as though the ixgbe driver being used by this tool wasn't built with support for the Intel x553 cards, regardless of which model or chipset is being emulated (no success with DS918+ or DS3615xs built in any combination of the supported chipsets (apollolake, bromolow, broadwell))

I've been successful with other Redpill scripts I've found on Github.
The only caveat is you need a working copy of ubuntu/debian linux to run the scripts. So maybe you'll need to spin up a VM or install WSL in Windows.

TL;DR: TinyCore-RedPill isn't foolproof and needs some work still. Try one of the scripts if you're having issues.
